app/src/main/kotlin/no/nav/k9punsj/omsorgspengerutbetaling/OmsorgspengerutbetalingSøknadDto.kt

```diff
@@ -0,0 +1,68 @@
+package no.nav.k9punsj.omsorgspengerutbetaling
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import com.fasterxml.jackson.module.kotlin.convertValue
+import no.nav.k9punsj.felles.FagsakYtelseType
+import no.nav.k9punsj.felles.dto.Mappe
+import no.nav.k9punsj.felles.dto.SøknadEntitet
+import no.nav.k9punsj.felles.dto.PeriodeDto
+import no.nav.k9punsj.felles.dto.hentUtJournalposter
+import no.nav.k9punsj.felles.DurationMapper.somDuration
+import no.nav.k9punsj.felles.DurationMapper.somTimerOgMinutter
+import no.nav.k9punsj.objectMapper
+import no.nav.k9punsj.pleiepengersyktbarn.PleiepengerSyktBarnSøknadDto
+import no.nav.k9punsj.pleiepengersyktbarn.PleiepengerSyktBarnSøknadDto.TimerOgMinutter.Companion.somTimerOgMinutterDto
+import java.time.LocalDate
+import java.time.LocalTime
+
+data class OmsorgspengerutbetalingSøknadDto(
+    val soeknadId: String,
+    val soekerId: String? = null,
+    @JsonFormat(pattern = "yyyy-MM-dd")
+    val mottattDato: LocalDate? = null,
+    @JsonFormat(pattern = "HH:mm")
+    val klokkeslett: LocalTime? = null,
+    val journalposter: List<String>? = null,
+    val organisasjonsnummer: String? = null,
+    val arbeidsforholdId: String? = null,
+    val fravaersperioder: List<FraværPeriode>? = null,
+    val harInfoSomIkkeKanPunsjes: Boolean? = null,
+    val harMedisinskeOpplysninger: Boolean? = null
+) {
+    data class FraværPeriode(
+        val periode: PeriodeDto,
+        val faktiskTidPrDag: String?,
+        val tidPrDag: PleiepengerSyktBarnSøknadDto.TimerOgMinutter? = faktiskTidPrDag?.somDuration()
```

Comment:
Denne funktionen skulle vart antingen felles eller kopieres in her.

Reply:
Refaktorert.